dungeon/test/contrib/systems/HealthSystemTest.java

```diff
@@ -18,7 +18,7 @@
 import org.mockito.Mockito;
 
 public class HealthSystemTest {
-  private static final IPath ANIMATION_PATH = new SimpleIPath("textures/test_hero");
+  private static final IPath ANIMATION_PATH = new SimpleIPath("character/blue_knight");
```

Comment:
diese änderung verstehe ich nicht. wieso ist die notwendig?

Reply:
`character/blue_knight` existiert so oder so in `dungeon`. Den `test_hero` hätte ich komplett aus `game` kopieren müssen.

Reply:
hmmm. 

- auf der einen seite finde ich einen reuse von ressourcen sinnvoll, d.h. dass man für den test nicht extra ressourcen bastelt, wenn man die in den sourcen sowieso hat (das spricht für deine änderung). 
- andererseits wurde hier der weg beschritten, die tests mit ihren ressourcen komplett unabhängig von den ressourcen der sourcen zu halten - d.h. konsequenterweise solltest du hier entsprechende kopien anlegen für den test. 

=> ich denke, wir sollten hier erstmal dem existierenden trampelpfad folgen und die die testressource kopieren. was denkst du?

---

ich glaube, das sollte später mal angepasst werden, so dass die tests die ressourcen der sourcen mitnutzen. sehe grad keinen richtigen sinn für extra test-ressourcen.

Reply:
Okay, dann mache ich mal Kopie-Pasta